FAQ.md

```diff
@@ -40,7 +40,12 @@ In a word: No.
 
 OAuth authentication requires the end-user to interact with a web browser, which can only be done client-side.
 
-It is possible to user OAuth authentication in a web application, but not using this `GoogleWebAuthorizationBroker` class.
+It is possible to use OAuth authentication for end users in a web application, but not using this `GoogleWebAuthorizationBroker` class.
+For these purposs, you can use one the ASP.NET integration packages we provide, for instance `Google.Apis.Auth.AspNetCore3`.
```

Comment:
porposs => purposes
one the => one of the

Reply:
Done!